nodes/src/nodes/agent_crewai/IInstance.py

```
@@ -42,8 +42,24 @@ def writeQuestions(self, question: Question):
+
+        # crewai.describe fan-out: only sub-agents (CrewDriver) respond — guarded by hasattr.
+        # OrchestratorDriver has no describe() so it silently falls through.
+        if isinstance(op, str) and op == 'crewai.describe' and hasattr(self.IGlobal.agent, 'describe'):
+            descriptor = self.IGlobal.agent.describe(self)
+            existing = getattr(param, 'agents', None)
+            if isinstance(existing, list):
+                existing.append(descriptor)
+                try:
+                    param.agents = existing
+                except Exception:
+                    pass
```

Ruff (0.15.7)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)